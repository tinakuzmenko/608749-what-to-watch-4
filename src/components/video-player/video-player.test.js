import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const videoData = {
  poster: `img/bg-the-grand-budapest-hotel.jpg`,
  preview: `https://upload.wikimedia.org/wikipedia/commons/7/72/Landwasserviadukt%2C_aerial_video.webm`,
};

describe(`VideoPlayer`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(<VideoPlayer
        muted
        isPlaying={false}
        source={videoData.preview}
        poster={videoData.poster}
      />, {
        createNodeMock: () => {
          return {};
        }
      })
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
