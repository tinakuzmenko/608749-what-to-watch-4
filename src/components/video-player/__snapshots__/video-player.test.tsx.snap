// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Should render correctly 1`] = `
<video
  height="175"
  width="280"
>
  Sorry, your browser doesn't support embedded videos.
</video>
`;
